### ISSUE TITLE
Cannot run flutter test integration on web, it fails with not supported for integration tests yet

### ISSUE THREAD

**Issue by iam-kj** [flutter/flutter#117955]
## Description
Flutter Integration tests were not allowed for web and causing exception, at the same time Flutter Drive works. are there any plans on bringing in support for Flutter Integration test? since we could see document talks about migration to Flutter Test Integration & not the Flutter Drive.

### Flutter Test Integration:

> `flutter test integration_test/test/sampleIT.dart --flavor qa --dart-define=environment=qa --dart-define=listenForOtp=false --no-sound-null-safety -d web-server --no-headless`

<img width="993" alt="Screenshot 2023-01-04 at 3 17 14 PM" src="https://user-images.githubusercontent.com/108999289/210531996-636e0711-84bd-4f43-9365-f72817941903.png">

### Flutter Drive Test:
> `flutter drive --driver=test_driver/integration_test.dart --target=integration_test/test/sampleIT.dart -d web-server --no-sound-null-safety --no-headless`

**Comment by exaby73** [comment]
Hello @iam-kj. Can you provide code sample so that I may verify the issue? Also provide the output to `flutter doctor -v`

**Comment by iam-kj** [comment]
Hi @exaby73 , you can make use of this demo project available at this repo: [Flutter demo project: (https://github.com/cedvdb/flutter_repros/tree/integration_test_failing_imports)

**Flutter Integration Test command example:** 
flutter test integration_test/test/sampleIT.dart --flavor qa --dart-define=environment=qa --dart-define=listenForOtp=false --no-sound-null-safety -d web-server --no-headless

**Flutter Doctor:**
<img width="1213" alt="Screenshot 2023-01-05 at 5 54 28 PM" src="https://user-images.githubusercontent.com/108999289/210780153-06e2d788-cfb6-4073-9b63-65b808f41513.png">

**Comment by exaby73** [comment]
## Triage report

I can reproduce the issue with the steps above. `flutter drive` seems to support web, but not `integration_test`. Labeling as a proposal

**Comment by ZacharyBohn** [comment]
If Flutter web is considered to be stable, then integration tests should be supported.

**Comment by Liortra** [comment]
@iam-kj @exaby73 
I tried to use flutter drive and add breakpoints in inspect(sources) but it's terrible, the dart files there are not really dart files, and putting a breakpoint puts me in an HTML file

**Comment by iam-kj** [comment]
is there a possibilities in future this Flutter Test start support for web? Documentation says Flutter Driver is deprecated & move to Integration test. but doesn't have enough documentation for web support.

**Comment by jaxnz** [comment]
> 

Yes, we are currently facing issue after issue with `flutter drive` for web. Is there any update on `integration_test` supporting web in the near future?

**Comment by pandoras-toolbox** [comment]
Wow, lot of activity in such a short time about it.

@iam-kj I will try to provide that code and the output of `flutter doctor -v` (I overlooked that in the issue template). I first need an answer from our security office if I am allowed to upload the full sample code to Gitlab from my company notebook for you to easier reproduce and see what my issues are.

**Comment by awsms** [comment]
> If Flutter web is considered to be stable, then integration tests should be supported.

this?

**Comment by c-seeger** [comment]
any update here?

**Comment by Executer13** [comment]
Any update?

**Comment by 9745108736** [comment]
any one get the update?

**Comment by swallow66** [comment]
Any updates on this is expected in some reasonable future?

**Comment by iam-kj** [comment]
It's been 2 years since this issue was created, still flutter test is not available yet for it's flutter website. 

Is that the flutter test wasn't considered by developers? If not how to test the application which was developed by flutter?

**Comment by jaxnz** [comment]
@AhmedAlyDev  - it works fine for everything except web. Please read the actual discussion

**Comment by AhmedAlyDev** [comment]
@jaxnz It works on web! make sure to follow this [](https://github.com/flutter/flutter/tree/main/packages/integration_test)
you could see this [example](https://github.com/flutter/flutter/tree/main/packages/integration_test/example) for inspiration.

**Comment by swallow66** [comment]
@AhmedAlyDev you use flutter drive in your example for web. But this thread is to have "flutter test" web support, which is still not in place as for now.

**Comment by minbar** [comment]
facing same issue

**Comment by ranjeetscience** [comment]
Any update on this . The example provided on flutter website test one file at one time for web . This is not something we expect from such widely used framework

**Comment by jaxnz** [comment]
@jonahwilliams - is there any plans to support this? Seems like a pretty heavy requirement
